data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/SqsOptions.java

```diff
@@ -19,6 +19,9 @@ public class SqsOptions {
     private static final Duration DEFAULT_VISIBILITY_TIMEOUT_SECONDS = Duration.ofSeconds(30);
     private static final Duration DEFAULT_WAIT_TIME_SECONDS = Duration.ofSeconds(20);
     private static final Duration DEFAULT_POLL_DELAY_SECONDS = Duration.ofSeconds(0);
+    private static final Duration DEFAULT_MAX_BACKOFF = Duration.ofSeconds(600);
```

Comment:
It might not be a bad idea to use `.ofMinutes(5)`. And then clarify this default in the documentation. Data Prepper pipeline configurations support configuring durations using either `600s` or `PT5M` to help make durations a little more readable.